benchmark-e2e/src/main/java/io/opentelemetry/e2ebenchmark/E2EAgentBenchmark.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.e2ebenchmark;
+
+import java.util.AbstractMap;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+public class E2EAgentBenchmark {
+  private static String app = System.getenv("APP_IMAGE");
```

Comment:
```suggestion
  private static final String APP_NAME = System.getenv("APP_IMAGE", "ghcr.io/open-telemetry/java-test-containers:smoke-springboot-jdk8-20201204.400701583");
```

Reply:
Sorry I did not understand this suggestion. `System.gentenv()` accepts 1 `String` param right?

Reply:
Oops - sorry `System.env().getOrDefault`